test/index.test.ts

```diff
@@ -169,6 +221,8 @@ describe('Parser', () => {
         y: 0.273,
         width: 0.156,
         height: 0.234,
+        idcRegionDefinitionId: 'crop-4875596e-49ca-3ba4-5965-6ba0850738ac',
```

Comment:
Can we remove these fields here, i.e. make them non-essential? (grep for `if (essentialOnly) {`)

Reply:
sorry, ignore this, I got confuse by some code folding here

Reply:
@AurelienLourot That's actually what I originally did, but then reverted for the following reason:

- With essentialOnly, the parser returns only IDC-related data. The image region definitions are IDC-related and can be very useful for controlling which region is displayed.
- Without essentialOnly the parser returns a lot of data that is not IDC related. Needing to fall back to the full data just in order to get a single additional IDC-related field feels a bit overkill.

It might make sense to introduce an additional level of granularity for returning IDC data, e.g. "minimum" that would return the minimum set of data that still makes sense for using IDC. Or alternatively being able to define which fields you want returned when calling the library. But this falls anyway outside this feature & PR.

Reply:
Makes sense, thanks :)